.github/workflows/release-please.yml

```diff
@@ -9,12 +9,16 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: git pull
```

Comment:
I'm assuming the `git pull` here was to get the entire history. I updated the checkout action to use `fetch-depth:0` which will fetch the entire history.

Reply:
Yes IIRC the newer version of this script does use `fetch-depth:0` for other actions